packages/atomic-react/README.md

```diff
@@ -295,90 +296,94 @@ import {AtomicSearchInterface} from '@coveo/atomic-react';
-
-dist
-.tarball: https://registry.npmjs.org/@coveo/atomic/-/atomic-1.108.2.tgz
-.shasum: 6e844073f55f10328b4b828c0190b8fe31403fbf
-.integrity: sha512-LhFR6k7NdGi8pYHmWOq3gM3rE0zeTKc6biTa1Z+ZKr5TEXIr97rpxi1HSnViQUowQYYCcY44dvsAYz2yyPvXTA==
-.unpackedSize: 87.1 MB
-
-dependencies:
-@coveo/bueno: 0.42.1                  @stencil/store: 1.5.0                 focus-visible: 5.2.0
-@coveo/headless: 1.103.3              coveo-styleguide: 9.34.4              i18next-http-backend: 1.4.1
-@popperjs/core: ^2.11.6               dayjs: 1.11.5                         i18next: 20.6.1
-@salesforce-ux/design-system: ^2.16.1 dompurify: 2.3.10                     stencil-inline-svg: 1.1.0
-@stencil/core: 2.17.3                 escape-html: 1.0.3                    ts-debounce: ^4.0.0
-
+$ npm view @coveo/atomic-react@latest version dependencies devDependencies
+
+version = '2.1.35'
+dependencies = { '@coveo/atomic': '2.19.16' }
+devDependencies = {
+  '@coveo/headless': '2.8.10',
+  '@rollup/plugin-commonjs': '^22.0.2',
+  '@rollup/plugin-node-resolve': '^14.1.0',
+  '@rollup/plugin-replace': '^4.0.0',
+  '@rollup/plugin-typescript': '^8.5.0',
+  '@types/node': '15.14.9',
+  '@types/react': '18.0.17',
+  '@types/react-dom': '18.0.6',
+  ncp: '2.0.0',
+  react: '18.2.0',
+  'react-dom': '18.2.0',
+  rollup: '^2.79.0',
+  'rollup-plugin-polyfill-node': '^0.10.2',
+  'rollup-plugin-terser': '^7.0.2'
+}
+peerDependencies = {
+  '@coveo/headless': '^2.0.0',
+  react: '>=18.0.0',
+  'react-dom': '>=18.0.0'
+}
 ```
 
-In the above example, you can see that `@coveo/atomic@1.108.2` has the dependency `@coveo/headless@1.103.3`.
+In the above example, you can see that of the time of this writing:
 
-To summarize, as of the time of this writing, the versions that we should keep in mind are the following:
+- The current latest version of `@coveo/atomic-react` is `2.1.35`.
+- The current latest version is using `@coveo/atomic` at version `2.19.16`.
+- The current latest version is using `@coveo/headless` at version `2.8.10`.
+- The current latest version is using `react`/`react-dom` at version 18 and above.
 
-- `@coveo/atomic-react` -> 1.23.11
-- `@coveo/atomic` -> 1.108.2
-- `@coveo/headless` -> 1.103.3
+Then, we need to add the required dependencies in our page. You can do that like so:
 
 ```html
 <head>
-  <!-- React and ReactDOM need to be included  -->
-  <script crossorigin src="https://unpkg.com/react@18/umd/react.production.min.js"></script>
-  <script crossorigin src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"></script>
+  <!-- React, ReactDOM and ReactDOMServer need to be included  -->
+  <script
+    crossorigin
+    src="https://unpkg.com/react@18/umd/react.production.min.js"
+  ></script>
+  <script
+    crossorigin
+    src="https://unpkg.com/react-dom@18/umd/react-dom.production.min.js"
+  ></script>
+  <script
+    crossorigin
+    src="https://unpkg.com/react-dom@18/umd/react-dom-server-legacy.browser.production.min.js"
```

Comment:
**Caveat**: `react-dom/umd/react-dom-server.browser.production.min.js` does not include the function we use in AtomicReact, which is `renderToString`. `react-dom/server.browser.js` does include everything we need, but it imports them via `require`, which throws an error.